src/dosbox.cpp

```diff
@@ -359,8 +367,8 @@ static void increase_ticks()
 			}
 		}
 
-		if (new_cycle_max < CPU_CYCLES_LOWER_LIMIT) {
-			new_cycle_max = CPU_CYCLES_LOWER_LIMIT;
+		if (new_cycle_max < auto_cpu_cycles_min) {
+			new_cycle_max = auto_cpu_cycles_min;
 		}
```

Comment:
Is there a better name for these variables?  I would expect the max to always be bigger than the min.